src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java

```diff
@@ -372,7 +373,14 @@ public void validate(MonitorableJob.Status status) {
                 );
             } else {
                 FeedSource fs = Persistence.feedSources.getById(this.feedSourceId);
-                SharedStopsValidator ssv = new SharedStopsValidator(fs.retrieveProject());
+
+                // Get feed_id from feed version... Really awful hack!
+                JDBCFetcher feedFetcher = new JDBCFetcher("feed_info", null);
+                Object gtfsFeedId = feedFetcher.getResults(this.namespace, null, null).get(0).get("feed_id");
+
+
+                String feedId = gtfsFeedId == null ? "" : gtfsFeedId.toString();
```

Comment:
The distinction is intentional. It is set to null when there is no value found and "" if we are confident there is no value. This might be me being a js developer shining through haha. looking for the null/undefined distinction